### PR TITLE
Add KWOK_MODE to configure several default cluster templates

### PR DESCRIPTION
**Pull request by wzshiming**
/kind feature
/kind bug

Fixes #36

### CODE REVIEW

File: pkg/kwokctl/vars/vars.go [hunk 1]
```diff
@@ -171,11 +171,28 @@ var (
 		return ""
 	}()
 
+	// Mode is several default parameter templates for clusters
+	Mode = getEnv("KWOK_MODE", "")
+
+	// ModeTiny efforts to reduce cluster configuration requirements
+	// Disables all alpha feature by default, as well as Bata feature that are not eventually GA
+	ModeTiny = "tiny"
```
Comment (Huang-Wei) [inline pkg/kwokctl/vars/vars.go:179]: "tiny" doesn't read well, can we change to another name? like `StableFeatureFateAndAPI`?

File: pkg/kwokctl/vars/vars.go [hunk 2]
```diff
@@ -171,11 +171,28 @@ var (
 		return ""
 	}()
 
+	// Mode is several default parameter templates for clusters
+	Mode = getEnv("KWOK_MODE", "")
```
Comment (Huang-Wei) [inline pkg/kwokctl/vars/vars.go:175]: So the default mode would be "", is it intentional?

Comment (wzshiming) [reply pkg/kwokctl/vars/vars.go:175]: Yes, No special configuration is done by default.

File: pkg/kwokctl/vars/vars.go (hunk 1 above)
Comment (wzshiming) [reply pkg/kwokctl/vars/vars.go]: Updated

File: pkg/kwokctl/vars/vars.go [hunk 3]
```diff
@@ -171,11 +171,28 @@ var (
...
+
+	// ModeStableFeatureGateAndAPI efforts to reduce cluster configuration requirements
```
Comment (yibozhuang) [inline pkg/kwokctl/vars/vars.go:177]:
```suggestion
	// ModeStableFeatureGateAndAPI is intended to reduce cluster configuration requirements
```

File: pkg/kwokctl/vars/vars.go [hunk 4]
```diff
@@ -171,11 +171,28 @@ var (
...
+	// Disables all alpha feature by default, as well as Bata feature that are not eventually GA
```
Comment (yibozhuang) [inline pkg/kwokctl/vars/vars.go:178]:
```suggestion
	// Disables all Alpha feature by default, as well as Beta feature that are not eventually GA
```

**Review by yibozhuang — APPROVED** [review]
/lgtm

**Pull request merged**